app/scripts/lib/channels/notifier.js

```diff
@@ -20,12 +20,12 @@ define(function (require, exports, module) {
     CHANGE_PASSWORD: {
       name: 'fxaccounts:change_password',
       schema: {
-        email: 'String',
-        keyFetchToken: 'String',
-        sessionToken: 'String',
-        uid: 'String',
-        unwrapBKey: 'String',
-        verified: 'Boolean'
+        email: Vat.email().required(),
+        keyFetchToken: Vat.string().required(),
```

Comment:
I was debating whether to add Vat entries for `keyFetchToken`, `sessionToken`, and `unwrapBKey` too, that way if we have to update the expected form, it can be done in one place. Seem reasonable?